### ISSUE TITLE
Update SWA batch norm statistics

### ISSUE THREAD

**Issue by growlix** [mosaicml/composer#624]
Currently, if `swa.swa_end` occurs after the final training epoch has commenced, the SWA/averaged model's batch norm statistics won't be updated, which leads to very poor performance. Proposed solution: SWA (and other algorithms that may require it) should set an internal flag indicating that an additional pass through the training data is necessary after the end of training but before the final validation pass.

**Comment by mvpatel2000** [comment]
Closing. Tracking elsewhere as low pri
